pipeline/db/migrations/20220411085604_add_pr_open_closed.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE benchmark_metadata
+ADD COLUMN pull_closed BOOLEAN DEFAULT FALSE;
```

Comment:
I think life can be tiny bit easier in the long run if we can have a `BOOLEAN NOT NULL` here (for branches, `pull_closed=false` sounds fine as we would get the expected "don't hide them" result)

(`NULL` is a bit of an "I-don't-know" state, so there's always a risk of interpreting it wrongly in the code later + it requires extra code to handle trinary logic!)

Reply:
Yes, I agree having a `NOT NULL` condition here would simplify things!